packages/@coorpacademy-components/src/organism/review-slide/test/on-press.test.tsx

```diff
@@ -18,4 +18,9 @@ test('should handle onPress on validate button', t => {
   const buttonValidate = getByTestId('slide-validate-button-0');
 
   fireEvent.press(buttonValidate, mockedGestureEvent);
+
+  // wait for onclick to be triggered
+  await new Promise(resolve => {
+    setTimeout(resolve, 50);
```

Comment:
on the code of the component the time is 20ms but here it is 50ms is there any reason for that ?